tests/datalab/test_datalab.py

```diff
@@ -19,20 +19,19 @@
 import io
 import os
 import pickle
+import timeit
+from pathlib import Path
 from unittest.mock import MagicMock, Mock, patch
-from cleanlab.datalab.datalab import Datalab
 
-from scipy.sparse import csr_matrix
-from sklearn.neighbors import NearestNeighbors
-from datasets.dataset_dict import DatasetDict
 import numpy as np
 import pandas as pd
-
-from pathlib import Path
-
 import pytest
-import timeit
+from datasets.dataset_dict import DatasetDict
+from scipy.sparse import csr_matrix
+from sklearn.neighbors import NearestNeighbors
```

Comment:
There needs to be a new unit test for the cleanvision integration. 
It needs to create a toy image dataset and verify cleanvision was actually able detect at least one type of issue (that you included in this toy image dataset).